drivers/windows/windows.go

```diff
@@ -168,6 +169,18 @@ func (d *driver) parseNetworkOptions(id string, genericOptions map[string]string
 			config.DNSSuffix = value
 		case DNSServers:
 			config.DNSServers = value
+		case MacPool:
+			config.MacPools = make([]hcsshim.MacPool, 0)
+			s := strings.Split(value, ",")
+			if len(s)%2 != 0 {
+				return nil, types.BadRequestErrorf("Invalid mac pool. You must specify both a start range and an end range")
+			}
+			for i := 0; i < len(s)-1; i += 2 {
```

Comment:
with the <, you don't really need the -1 in the upper bound of the loop (also because you already checked that the number of elements is even).
if len(s) == 2 in the first loop i = 0, then i = 2 < 2 is already false